tests/interfaces/test_jax.py

```diff
@@ -723,6 +723,31 @@ def cost(a, cache):
         res = jax.jacobian(cost)(params, cache=None)
         assert res.shape == (2, 1)
 
+    def test_multi_tape_fwd(self, execute_kwargs):

```

Comment:
Should this test not also have different number of parameters, to check the `assert` above?

Reply:
It can have diff number of params, just updated. :+1: